diff --git a/src/gui/folderman.cpp b/src/gui/folderman.cpp
index 18759a6..0fe8b19 100644
--- a/src/gui/folderman.cpp
+++ b/src/gui/folderman.cpp
@@ -672,7 +672,7 @@ void FolderMan::slotStartScheduledFolderSync()
     }
 
     // Find the first folder in the queue that can be synced.
-    Folder* f = nullptr;
+    Folder* f = 0; // nullptr;
     while( !_scheduleQueue.isEmpty() ) {
         f = _scheduleQueue.dequeue();
         Q_ASSERT(f);
