--- a/src/libsync/account.cpp	2015-03-16 14:28:58.000000000 +0000
+++ b/src/libsync/account.cpp	2015-03-30 17:09:10.383648721 +0000
@@ -295,7 +295,7 @@
 QNetworkReply *Account::getRequest(const QUrl &url)
 {
     QNetworkRequest request(url);
-    request.setSslConfiguration(this->createSslConfig());
+    //request.setSslConfiguration(this->createSslConfig());
     return _am->get(request);
 }
 
@@ -307,7 +307,7 @@
 QNetworkReply *Account::davRequest(const QByteArray &verb, const QUrl &url, QNetworkRequest req, QIODevice *data)
 {
     req.setUrl(url);
-    req.setSslConfiguration(this->createSslConfig());
+    // req.setSslConfiguration(this->createSslConfig());
     return _am->sendCustomRequest(req, verb, data);
 }
 
